neuralmonkey/attention/scaled_dot_product.py

```diff
@@ -69,25 +197,23 @@ def attention(self,
                   loop_state: MultiHeadLoopStateTA,
                   step: tf.Tensor) -> Tuple[tf.Tensor, MultiHeadLoopStateTA]:
 
-        if self.n_heads == 1:
-            context, weights = self.attention_single_head(
-                query, self.attention_keys, self.attention_values)
-            head_weights = [weights]
-        else:
-            # project query, keys and vals: [batch, rnn] to [batch, rnn2]
-            head_contexts, head_weights = zip(*[
-                self.attention_single_head(
-                    tf.layers.dense(query, self._head_dim,
-                                    name="query_proj_head{}".format(i)),
-                    tf.layers.dense(self.attention_keys, self._head_dim,
-                                    name="keys_proj_head{}".format(i)),
-                    tf.layers.dense(self.attention_values, self._head_dim,
-                                    name="values_proj_head{}".format(i)))
-                for i in range(self.n_heads)])
-
-            context = tf.layers.dense(
-                tf.concat(head_contexts, -1), self._dimension,
-                name="output_proj")
+        # transform (batch, query_size) to (batch, 1, query_size)
```

Comment:
convert these comments to docstring
add Args and Returns, say that this function is a API-conforming wrapper to the `attention` function defined above in the module.